submissions/Eugene-CG/DOM/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Side-Menu</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="./mediaquery.css" />
+    <script src="./index.js" defer></script>
+  </head>
+  <body>
+    <div class="delegate-events-container">
+      <div class="popup">
+        <ul class="popup__list">
+          <li class="popup__list-item first">First</li>
```

Comment:
For all interactive elements, you should use corresponding HTML tags. Avoid using HTML tags for static elements as interactive.

Reply:
dont understand